redox/processor_neworder.go

```diff
@@ -919,7 +921,6 @@ func (c CreateAccountEnableReports) GetMatchRequest() clinics.EHRMatchRequest {
 	return request
```

Comment:
You should check here if you have any observations and return a `nil` flowsheet if there aren't any. This should be behind a feature flag.